addons/purchase_requisition/models/product.py

```diff
@@ -16,7 +16,7 @@ class ProductProduct(models.Model):
 
     def _prepare_sellers(self, params=False):
         sellers = super(ProductProduct, self)._prepare_sellers(params=params)
-        if params and params.get('order_id'):
+        if params and params.get('order_id') and params['order_id']._fields.get("requisition_id"):
```

Comment:
Why this change? If an order is passed in the param and the order doesn't have a requisition_id then we want to make sure that we don't select a seller that DOES have purchase_requisition_id, otherwise we may incorrectly select a seller that we shouldn't

Reply:
Since i'm passing an account.move as the order, account.move doesn't have a requisition_id and it causes the thing to crash, i don't think we want to add the requisition_id to our model that's why i've done this change. If you think there is another way i would gladly take it :smile:

Reply:
Ah sorry, I misread what you were doing :see_no_evil: please disregard my comment

Reply:
@malb-odoo But does it make sense at the first place to call `_select_seller` while passing an `account.move` as the order ?

Reply:
@svs-odoo Actually yes it does, in case of a bill, we can have multiple price depending on the seller